src/tasks/tasks.py

```diff
@@ -1558,43 +1558,63 @@ def get_metrics(self, reset=False):
         acc = self.scorer1.get_metric(reset)
         return {'accuracy': acc}
 
-    def process_split(self, split, indexers) -> Iterable[Type[Instance]]:
```

Comment:
Are there any other tasks that subclass TaggingTask and might rely on this?

Reply:
Nope!